src/FizzBuzz.java

```diff
@@ -1,17 +1,25 @@
+import java.util.ArrayList;
+
 public class FizzBuzz {
     public static void main(String[] args) throws Exception {
 
+        ArrayList<String> result = new ArrayList<String>();

```

Comment:
Nice this is a pretty good approach! For performance and memory reasons though, you might want to use a "StringBuilder" https://www.geeksforgeeks.org/stringbuilder-class-in-java-with-examples/

Since you're never doing any special array functions (like sorting, splicing etc)